R/mod_chartsTab.R

```diff
@@ -40,16 +40,22 @@ chartsTab <- function(input, output, session, chart, data, mapping){
   })
 
   # Draw the chart
-  if(chart$type=="module"){
-    callModule(chart$functions$main, "chart-wrap", params)
-  }else{
-    output[["chart-wrap"]] <- chart$functions$server(
+  if(chart$type=="module") {
+    module_output <- callModule(
+        chart$functions$main,
+        "chart-wrap",
+        params,
+        module_outputs
+    )
+  } else {
+    module_output <- output[["chart-wrap"]] <- chart$functions$server(
```

Comment:
Is anything actually being returned here for non-shiny charts? I guess in theory the `main` function could return something, but is this just NULL for now?

Reply:
Right, nothing now...  I can imagine returning a filtered/transformed dataset or a population subset.